rocky/crisis_room/migrations/0001_initial.py

```diff
@@ -0,0 +1,71 @@
+# Generated by Django 5.0.9 on 2024-12-18 11:29
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = [("tools", "0044_alter_organization_options")]
+
+    operations = [
+        migrations.CreateModel(
+            name="Dashboard",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("name", models.CharField(max_length=126)),
+                (
+                    "organization",
+                    models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, to="tools.organization"),
+                ),
+            ],
+            options={"unique_together": {("name", "organization")}},
+        ),
+        migrations.CreateModel(
+            name="DashboardData",
+            fields=[
+                ("id", models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
+                ("recipe", models.CharField(max_length=126)),
+                ("template", models.CharField(blank=True, default="findings_report/report.html", max_length=126)),
+                (
+                    "position",
+                    models.IntegerField(
+                        blank=True,
+                        default=1,
+                        help_text="Where on the dashboard do you want to show the data? Position 1 is the most top level and the max position is 16.",
+                        validators=[
+                            django.core.validators.MinValueValidator(1),
+                            django.core.validators.MaxValueValidator(16),
+                        ],
+                    ),
+                ),
+                (
+                    "display_in_crisis_room",
+                    models.BooleanField(
+                        default=False, help_text="Will be displayed on the general crisis room, for all organizations."
+                    ),
+                ),
+                (
+                    "display_in_dashboard",
+                    models.BooleanField(
+                        default=False, help_text="Will be displayed on a single organization dashboard"
+                    ),
+                ),
+                (
+                    "findings_dashboard",
+                    models.BooleanField(
+                        default=False, help_text="Will be displayed on the findings dashboard for all organizations"
+                    ),
+                ),
+                (
+                    "dashboard",
+                    models.ForeignKey(
+                        null=True, on_delete=django.db.models.deletion.SET_NULL, to="crisis_room.dashboard"
```

Comment:
This can probably be a cascade, where the dashboarddata gets deleted when the dashboard gets deleted. Not just set to null. Null values are not needed.